crates/store/re_chunk/src/transport.rs

```diff
@@ -224,10 +218,11 @@ impl Chunk {
             components
         };
 
+        let is_sorted_by_row_id = batch.chunk_schema().row_id_column().is_sorted;
         let mut res = Self::new(
-            id,
-            entity_path,
-            is_sorted.then_some(true),
+            batch.chunk_id(),
+            batch.entity_path().clone(),
+            is_sorted_by_row_id.then_some(true),
```

Comment:
The doc below says 
```
`false` means either "unsorted" or "unknown".
```
but this seems to say that the mere presence of a value means `true`? which is it?

Reply:
Right now we only encode the knowledge that is it sorted if it is sorted… maybe we should change that though.